frontend/src/app/core/global_search/input/global-search-input.component.html

```diff
@@ -44,10 +45,10 @@
       </div>
       <ng-template #workPackageItemTemplate>
         <a tabindex="-1"
-          class="global-search--option"
-          [href]="wpPath(item.id)"
-          (click)="redirectToWp(item.id, $event)"
-          style="line-height: 1">
+           class="global-search--option"
+           [href]="wpPath(item.id)"
+           (click)="redirectToWp(item.id, $event)"
```

Comment:
this double stuff is weird. is this a link or a button?

Reply:
The weirdness comes from ng-select handling the clicks to select stuff and cancelling events on the go. So it has an href to properly handle clicking with modifiers (which ng-select doesn't capture) and clicking to send to the wpPath.